Doc/library/re.rst

```diff
@@ -1565,18 +1565,22 @@ search() vs. match()
 Python offers two different primitive operations based on regular expressions:
```

Comment:
Maybe continue the style of this paragraph, so instead something like 'Python offers a few primitive ...' then '... re.fullmatch checks that the whole string is a match ...' (can change this wording up a bit too). I also think the `None` case can omitted since it wasn't mentioned in the previous two cases either.

Reply:
Done sir